flask_elasticsearch.py

```diff
@@ -33,7 +33,11 @@ def __getattr__(self, item):
         ctx = stack.top
         if ctx is not None:
             if not hasattr(ctx, 'elasticsearch'):
-                ctx.elasticsearch = Elasticsearch(hosts=[ctx.app.config.get('ELASTICSEARCH_HOST')],
+                if type(ctx.app.config.get('ELASTICSEARCH_HOST')) is str:
+                    hosts = [ctx.app.config.get('ELASTICSEARCH_HOST')]
+                elif type(ctx.app.config.get('ELASTICSEARCH_HOST')) is list:
```

Comment:
Same here, perhaps `isinstance(host, list)` so that it can handle subclasses of list as well